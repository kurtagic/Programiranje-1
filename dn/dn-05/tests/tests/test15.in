83
7 50
7 64
7 82
7 55
7 80
7 73
7 58
7 63
7 45
7 71
7 44
7 58
7 53
7 64
7 59
7 40
7 51
7 43
7 66
7 82
7 50
7 67
7 49
7 67
7 66
7 58
7 40
7 78
7 43
7 65
7 51
7 45
7 59
7 62
7 79
7 44
7 82
7 43
7 45
7 48
7 66
7 54
7 57
7 71
7 44
7 53
7 63
7 66
7 66
7 72
7 57
7 68
7 45
7 78
7 82
7 53
7 50
7 75
7 74
7 55
7 51
7 63
7 51
7 74
7 62
7 70
7 47
7 40
7 76
7 63
7 47
7 57
7 67
7 54
7 64
7 82
7 51
7 73
7 61
7 74
7 72
7 70
7 43
