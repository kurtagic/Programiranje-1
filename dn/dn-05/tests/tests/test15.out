40: 9
41: 7
43: 20
44: 18
45: 18
46: 16
47: 19
48: 17
49: 11
50: 11
51: 18
52: 16
53: 12
54: 13
55: 11
56: 5
57: 19
58: 17
59: 11
60: 9
61: 19
62: 17
63: 17
64: 18
65: 16
66: 20
67: 21
68: 19
69: 6
70: 12
71: 10
72: 15
73: 13
74: 13
75: 11
76: 4
77: 2
78: 9
79: 7
80: 4
81: 1
82: 15
83: 13
